app.rb

```diff
@@ -0,0 +1,55 @@
+require 'sinatra/base'
+require 'sinatra/activerecord'
+require './config/environment'
+
+# Global for verbose output
+# $verbose = true
+
+class Peep < ActiveRecord::Base 
+
+end
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  set :session_secret, ENV.fetch('SESSION_SECRET') { SecureRandom.hex(64) }
+
+  before '/' do
+    verbose_output(request) if $verbose
+  end
+
+  get '/' do
```

Comment:
I feel like this path could have a clearer definition, is this an index for users, peeps or something else?